src/modules/esl-share/core/esl-share-button.less

```diff
@@ -0,0 +1,26 @@
+esl-share-button {
+  display: inline-block;
+  width: 40px;
+  height: 40px;
+  cursor: pointer;
+
+  &:hover {
+    opacity: .6;
+  }
+
+  &[unavailable] {
+    display: none;
+  }
+
+  .esl-share-icon {
+    display: inline-block;
+    overflow: hidden;
+    background-color: #0097e7; // TODO: consider creating common variables.less file
+    width: 100%;
+    height: 100%;
+  }
+
+  path, circle {
```

Comment:
It doesn't look like a good idea at all to modify specific parts of SVG as a base style in the library (we don't know anything about the icons structure here, so we should modify and define styles abstractly or in common known way)

Reply:
moved to localdev styles